core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ambari/AmbariDecommissionTimeCalculator.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.cloudbreak.service.cluster.ambari;
+
+import static com.sequenceiq.cloudbreak.api.model.Status.AVAILABLE;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.core.flow2.stack.FlowMessageService;
+import com.sequenceiq.cloudbreak.core.flow2.stack.Msg;
+import com.sequenceiq.cloudbreak.domain.Template;
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.host.HostMetadata;
+
+@Component
+public class AmbariDecommissionTimeCalculator {
+
+    private static final int DECOMMISSIONING_MAGIC_NUMBER = 12;
+
+    private static final int SECONDS_PER_MINUTE = 60;
```

Comment:
Code duplication with `ClusterToClusterResponseConverter.SECONDS_PER_MINUTE`

Reply:
I think either we should create a class for providing these kind of constants in a common module or import it from a library, but importing it from a not related converter class `ClusterToClusterResponseConverter.SECONDS_PER_MINUTE` is not a good idea.